### PR TITLE
Added app-only DelegatedPermissionGrant.RW.All

### PR DESCRIPTION
**Pull request by Jackson-Woods**
Added the app-only DelegatedPermissionGrant.ReadWrite.All permission to all v1.0 and beta oauth2PermissionGrant reference pages.

### CODE REVIEW

**Review by psignoret — APPROVED** [review]
LGTM. Thanks!

File: api-reference/beta/api/oauth2permissiongrant-delta.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ One of the following permissions is required to call this API. To learn more, in
 |:--------------------|:---------------------------------------------------------|
 |Delegated (work or school account) | Directory.Read.All, Directory.ReadWrite.All    |
```
Comment (FaithOmbongi) [inline api-reference/beta/api/oauth2permissiongrant-delta.md:25]: I think `DelegatedPermissionGrant.ReadWrite.All` should be added here too?
```suggestion
|Delegated (work or school account) | Directory.Read.All, DelegatedPermissionGrant.ReadWrite.All, Directory.ReadWrite.All    |
```

File: api-reference/beta/api/oauth2permissiongrant-delta.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ One of the following permissions is required to call this API. To learn more, in
...
 |Delegated (personal Microsoft account) | Not supported.    |
-|oauth2permissiongrant | Directory.Read.All, Directory.ReadWrite.All |
+|oauth2permissiongrant | Directory.Read.All, DelegatedPermissionGrant.ReadWrite.All, Directory.ReadWrite.All |
```
Comment (FaithOmbongi) [inline api-reference/beta/api/oauth2permissiongrant-delta.md:27]:
```suggestion
| Application | Directory.Read.All, DelegatedPermissionGrant.ReadWrite.All, Directory.ReadWrite.All |
```

File: api-reference/v1.0/api/oauth2permissiongrant-delta.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ One of the following permissions is required to call this API. To learn more, in
 |:--------------------|:---------------------------------------------------------|
 |Delegated (work or school account) | Directory.Read.All, Directory.ReadWrite.All    |
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/oauth2permissiongrant-delta.md:23]: Missing permission?
```suggestion
|Delegated (work or school account) | Directory.Read.All, DelegatedPermissionGrant.ReadWrite.All, Directory.ReadWrite.All    |
```

**Review by FaithOmbongi — APPROVED** [review]
Thank you @Jackson-Woods for raising this fix. I've left suggestions for fixes - 2 possible missing instances and an incorrectly named cell. Approved with suggestions.

**Pull request merged**